.github/workflows/test-accessibility-alt-text-bot.yml

```diff
@@ -5,11 +5,11 @@ on:
   pull_request:
     types: [opened, edited]
   issue_comment:
```

Comment:
hah it always trips me up that `issue_comment` is the event trigger for both an issue comment and PR comment!

Reply:
1000%